webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -203,7 +203,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
         ) : error ? (
           <Error />
         ) : (
-          <>
+          <div>
```

Comment:
do we need this div?

Reply:
Double scroll bars on log view should be fixed